prow/e2e-suite.sh

```diff
@@ -30,11 +30,27 @@ set -u
 # Print commands
 set -x
 
-TESTS_TARGETS="e2e_simple e2e_mixer e2e_bookinfo e2e_upgrade"
+TEST_TARGETS=(e2e_simple e2e_mixer e2e_bookinfo e2e_upgrade)
 SINGLE_MODE=false
-E2E_ARGS=()
 
-source ${ROOT}/prow/lib.sh
+# Check https://github.com/istio/test-infra/blob/master/boskos/configs.yaml
```

Comment:
url is wrong

Reply:
Yes the PR is still pending review.

Reply:
PR is pending. https://github.com/istio/test-infra/pull/690